docker/docker-compose.dev.yml

```
@@ -129,6 +129,10 @@ services:
 
   # Standalone database for development
   database:
+    security_opt:
+      - "no-new-privileges:true"
+    security_opt:
```

YAMLlint (1.37.1)

[error] duplication of key "security_opt" in mapping

(key-duplicates)